templates/entry/_options.html.twig

```diff
@@ -112,7 +92,8 @@
             <button
                     aria-label="{{ 'filter_by_type'|trans }}"
                     title="{{ 'filter_by_type'|trans }}"><i
-                        class="fa-solid fa-filter"></i> {{ 'filter_by_type'|trans }}</button>
```

Comment:
I removed it saying "Filter by time" "Filter by type" "Change view". They were doing it a font-size 0 so it was never displayed visually. I _am_ wondering if this was for screen readers; but according to google, which may or may not be accurate as it has changed a lot over the years, setting to font size 0 isn't recommended (just as display none would also be skipped over). If it is an issue, we'd have it in lots of other places like federation / settings sidebar as those also don't have any text hidden on them